warnings_config.yml

```diff
@@ -0,0 +1,20 @@
+sphinx:
+  enabled: true
+  min: 0
+  max: 0
+  exclude:
+    - 'WARNING: Connection failed: HTTPSConnectionPool\(host=.+, port=\d+\): Max retries exceeded with url: /api/v2/[\w/?=&\\]+ \(Caused by NameResolutionError\(\"<urllib3\.connection\.HTTPSConnection object at [0-9a-z]+>: Failed to resolve .+ \(\[Errno -2\] Name or service not known\)\"\)\)'
+    - 'WARNING: CID \d+: Could not find item ID .+ in traceability collection.'
+    - 'WARNING: cannot cache unpickable configuration value: .traceability_attributes_sort. \(because it contains a function, class, or module object\)'
```

Comment:
Where does this warning come from? I don't remember seeing it before.

Reply:
It was something from the traceability extension. Deleted in c28f72bd687ba701a2c0ae2d0bf9436a069da7c5

Reply:
But I still need to solve it... https://github.com/melexis/sphinx-coverity-extension/actions/runs/10340653524/job/28621550305?pr=74#step:6:69

Reply:
I think this is a new warning introduced in Sphinx==7.3.0. We don't use `sphinx-build`'s cache functionality but someone else might. This warning should be addressed in https://github.com/melexis/sphinx-traceability-extension/ and should be excluded here in the meantime.

FYI https://github.com/sphinx-doc/sphinx/issues/12300

Reply:
Reverted commit c28f72bd687ba701a2c0ae2d0bf9436a069da7c5